src/http-auth-interceptor.js

```diff
@@ -125,6 +129,13 @@
           retryHttpRequest(updater(buffer[i].config), buffer[i].deferred);
         }
         buffer = [];
+      },
+
+      /**
+       * Check whether the buffer is empty
+       */
+      bufferEmpty: function() {
```

Comment:
Maybe rename the `bufferEmpty` to `isEmpty` and you could also get rid of that useless comment. What else the `httpBuffer.isEmpty()` can possibly do?